session/bootstrap_test.go

```diff
@@ -1945,3 +1945,30 @@ func TestTiDBStoreBatchSizeUpgradeFrom650To660(t *testing.T) {
 		}()
 	}
 }
+
+func TestTiDBUpgradeToVer136(t *testing.T) {
+	store, _ := createStoreAndBootstrap(t)
+	defer func() {
+		require.NoError(t, store.Close())
+	}()
+
+	ver135 := version135
+	seV135 := createSessionAndSetID(t, store)
+	txn, err := store.Begin()
+	require.NoError(t, err)
+	m := meta.NewMeta(txn)
+	err = m.FinishBootstrap(int64(ver135))
+	require.NoError(t, err)
+	mustExec(t, seV135, fmt.Sprintf("update mysql.tidb set variable_value=%d where variable_name='tidb_server_version'", ver135))
+	err = txn.Commit(context.Background())
+	require.NoError(t, err)
+
+	unsetStoreBootstrapped(store.UUID())
+	ver, err := getBootstrapVersion(seV135)
```

Comment:
This test doesn't test `v136`? I only see `v135`.

Reply:
v136 is upgraded in L1971

Reply:
But it doesn't feel like the v136 version was actually tested. I don't know if it was actually upgraded